GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/Controller/GameDataController.cs

```diff
@@ -0,0 +1,90 @@
+#define ONLINE
+//#define OFFLINE
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using static Utils.Server;
+
+
+public class GameDataController
```

Comment:
Is this a game object? If so, it should extend MonoBehaviour.

Reply:
Is not a game object, is more a class that will handle the game data of the game for saving it online and offline.